src/index.ts

```diff
@@ -94,8 +93,10 @@ const detect = async ({
 
 export { detect };
 
-export function getNpmVersion(pm: PM) {
-  return execa(pm || "npm", ["--version"]).then((res) => res.stdout);
+export async function getNpmVersion(pm: PM) {
+  const res = await x(pm || "npm", ["--version"]);
+
+  return res.stdout;
```

Comment:
In case we settle on using `tinyexec` instead of the solution proposed by Ben, we should consider calling `.trim()` here, as I think `execa` trims the trailing whitespace by default (found this out when some tests in other projects broke when I changed from `execa` to `tinyexec` and didn't call `.trim()`).

Reply:
Good catch! It's probably worth us opening an issue in the tinyexec repo to at least document this too (if you don't get chance, I'll open it)

Reply:
Opened https://github.com/tinylibs/tinyexec/issues/36. 🙂